python/lsst/ts/ess/sensors/mock/mock_temperature_sensor.py

```diff
@@ -0,0 +1,116 @@
+# This file is part of ts_ess_sensors.
+#
+# Developed for the Vera C. Rubin Observatory Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+__all__ = ["MockTemperatureSensor", "MIN_TEMP", "MAX_TEMP"]
+
+import asyncio
+import logging
+import random
+import time
+
+import numpy as np
+
+from lsst.ts.ess.sensors.sel_temperature_reader import DELIMITER
+
+# Minimum and maximum temperatures (deg_C) for creating random sensor data.
+MIN_TEMP = 18.0
+MAX_TEMP = 30.0
+
+
+class MockTemperatureSensor:
+    """Mock Temperature Sensor."""
```

Comment:
Please document the constructor arguments. I have no idea what count_offset does (and reading the unit test that sets it to 1 was not enlightening).

Reply:
Also in some future ticket you might consider making MIN_TEMP and MAX_TEMP class attributes. That way it's a bit easier to see what they control -- in particular you could hoist `MockTemperatureSensor` into the namespace `lsst.ts.ess.sensors` without worrying users about what `lsst.ts.ess.sensors.MIN/MAX_TEMP` mean.